source/MRMesh/MRRegionBoundary.cpp

```diff
@@ -412,15 +412,7 @@ static VertBitSet getIncidentVerts_( const MeshTopology & topology, const Undire
 FaceBitSet getIncidentFaces( const MeshTopology & topology, const UndirectedEdgeBitSet & edges )
 {
     MR_TIMER
-    FaceBitSet res( topology.faceSize() );
-    for ( auto ue : edges )
-    {
-        if ( auto l = topology.left( ue ) )
-            res.set( l );
-        if ( auto r = topology.right( ue ) )
-            res.set( r );
-    } 
-    return res;
+    return getIncidentFaces( topology, getIncidentVerts_( topology, edges ) );
```

Comment:
New implementation is right, but let us preserve the old one as function `getNeighborFaces`